pkg/apis/proto/mvtxdaemon/mvtxdaemon.proto

```diff
@@ -37,11 +37,30 @@ message GetMonoVertexMetricsResponse {
   MonoVertexMetrics metrics = 1;
 }
 
+// MonoVertexStatus is used to provide information about the mono vertex status.
+message MonoVertexStatus {
+  string status = 1;
+  string message = 2;
+  string code = 3;
+}
+
+message GetMonoVertexStatusRequest {
+  string monovertex = 1;
```

Comment:
Using `google.protobuf.Empty` for the request is good enough.

Reply:
Sure! For now I had copied over the pipeline template and was just adding the implementation. 
We wouldn't need it the pipeline API as well I checked. 
I'll update for both

Reply:
```
  // GetPipelineWatermarks return the watermark of the given pipeline
  rpc GetPipelineWatermarks (GetPipelineWatermarksRequest) returns (GetPipelineWatermarksResponse) {
    option (google.api.http).get = "/api/v1/pipelines/{pipeline}/watermarks";
  };

  rpc GetPipelineStatus (GetPipelineStatusRequest) returns (GetPipelineStatusResponse) {
    option (google.api.http).get = "/api/v1/pipelines/{pipeline}/status";
  };
```
Actually, we won't need for these 2 pipeline level APIs to daemon as well

Reply:
This is right.

Reply:
@whynowy 
Sorry just to clarify, you mean that you are good with the current implementation or are you good to change them to empty?
We don't use the request so I believe we can update to empty

Reply:
Let's use empty for MonoVertex. For Pipeline, we need to do several iterations if we want to correct it, leave it for now.

Reply:
Sure, Will create an issue to track that and refactor for later!